src/test/outmdsd/testqueue.cc

```diff
@@ -108,23 +108,19 @@ WaitEmptyQueue(
     std::promise<void>& masterReady,
     std::promise<void>& threadReady,
     ConcurrentQueue<int>* q,
-    uint32_t minRunTimeMS)
+    std::atomic<bool>& stopQueue)
 {
     threadReady.set_value();
     masterReady.get_future().wait();
 
-    auto startTime = std::chrono::system_clock::now();
     const int origVal = -123;
     int actual = origVal;
     q->wait_and_pop(actual);
 
     // nothing should be popped, so value shouldn't be changed
     BOOST_CHECK_EQUAL(origVal, actual);
-
-    auto endTime = std::chrono::system_clock::now();
-    auto runTimeMS = static_cast<uint32_t>((endTime-startTime)/std::chrono::milliseconds(1));
-    BOOST_CHECK_GE(runTimeMS, minRunTimeMS);
-    BOOST_TEST_MESSAGE("queue wait_and_pop item=" << actual << "; runtime: " << runTimeMS);
+    // stopQueue must be set to be true by now
+    BOOST_CHECK_EQUAL(true, stopQueue.load());
```

Comment:
agree. BOOST_CHECK() is better and cleaner. updated all Boolean check to BOOST_CHECK().